packages/api/src/external/commonwell/patient-conversion.ts

```diff
@@ -7,14 +7,17 @@ import {
   Patient as CommonwellPatient,
   Person as CommonwellPerson,
   StrongId,
+  GenderCodes,
 } from "@metriport/commonwell-sdk";
 import { addOidPrefix, driversLicenseURIs } from "@metriport/core/domain/oid";
 import { GenderAtBirth, Patient, splitName } from "@metriport/core/domain/patient";
 import { normalizePhoneNumber } from "@metriport/shared";
 
 const genderMapping: { [k in GenderAtBirth]: string } = {
-  F: "F",
-  M: "M",
+  F: GenderCodes.F,
+  M: GenderCodes.M,
+  O: GenderCodes.UN,
+  U: GenderCodes.UNK,
```

Comment:
MetriportGender to IheGender for CW

Reply:
Since we're touching this, could we fix the type of `genderMapping` so its `{ [k in GenderAtBirth]: GenderCodes }`, please?

Then, we can update the return type of `mapGenderAtBirthToCw` to return `GenderCodes`.